nemo_text_processing/inverse_text_normalization/pt/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
Could you please update the year for all new files 2021 -> 2022?

Reply:
@ekmb the style and copyright year have been fix